.rustfmt.toml

```diff
@@ -0,0 +1 @@
+use_field_init_shorthand = true
```

Comment:
Can this be deleted? I don't think this needs to be long-lived in this crate.

Reply:
It absolutely can be deleted. However this config would prevent ever having to have this conversation again...